stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -4468,7 +4469,10 @@ internal constructor(
         @JvmOverloads
         public fun handlePushMessage(pushMessage: PushMessage) {
             ensureClientInitialized().run {
-                if (!config.notificationConfig.ignorePushMessagesWhenUserOnline || !isSocketConnected()) {
+                val shouldHandle = !config.notificationConfig.ignorePushMessagesWhenUserOnline ||
+                    !isSocketConnected() ||
+                    pushMessage.type == NotificationType.NOTIFICATION_REMINDER_DUE
```

Comment:
Why this new type is ignored?

Reply:
The class `ChatNotifications` already is invoked when a new message is received from the WS to allow our customers to show a notification or not (Maybe depending if they are on the current channel or not), we should follow the same approach.
I know this PN Type is time-sensitive, and just becouse that if the user is connected to the WS we should rely on it instead on the PN that could fail.